File: 프로그래머스/3/12938. 최고의 집합/최고의 집합.py
def solution(n, s):
    answer = []
    
    if s//n==0:
        return [-1]
    
    for _ in range(n-s%n):
        answer.append(s//n)
        
    for _ in range(s%n):
        answer.append(s//n+1)

    
    return answer

File: 프로그래머스/3/12938. 최고의 집합/최고의 집합.java
import java.util.*;
class Solution {
    public int[] solution(int n, int s) {
        int[] answer = new int[n];
        
        if(s<n) return new int[]{-1};
        
        int a = s/n;
        int b = s%n;
        
        Arrays.fill(answer, a);
        for(int i=n-1;i>=n-b;i--){
            answer[i]++;
        }
        
        
        return answer;
    }
}